src/app/pages/wallets/wallet-details/pages-wallet-details.types.ts

```diff
@@ -0,0 +1,18 @@
+export interface WalletsDetails {
+  id: number;
+  name: string;
+  transaction: WalletTransactionDetails[];
+}
+
+export interface WalletTransactionDetails {
+  id: number;
+  date: string;
+  description: string;
+  amount: number;
+}
+
+export enum WalletTransactionType {
+  AllTransaction = "ALL_TRANSACTIONS",
```

Comment:
This is actually not a transaction type. Transactions can only have type "Income" and "Expense". Those are more like transaction filter values.